_plugins/shuffle.rb

```diff
@@ -0,0 +1,9 @@
+module Jekyll
```

Comment:
I'm not sure whether plugins works on github-pages. They have a [list of accepted plugins](https://help.github.com/articles/adding-jekyll-plugins-to-a-github-pages-site/) for some reason...  Probably a js function would be more safe...